bin/disco_vpc_ui.py

```diff
@@ -8,7 +8,11 @@
 import argparse
 import sys
 
-from disco_aws_automation import DiscoVPC
+from disco_aws_automation import (
```

Comment:
If we have a lot of imports and thus the need to have multiple lines, this way would save us from having to use line continuation characters `\`. I am just trying to be consistent here event we only have two imports.